### PR TITLE
Add doc about manupulate history

### LINKED ISSUE #821: browserHistory.push("/") don't work as expected
My app needed the locale feature so I included `react-intl` to my app.
When I dispatch my `setLocale` action, I need to redirect to my homepage and "refresh" the content because I build my content with `this.store.intl.locale` and it has to update.

So I did

```
import {browserHistory} from "react-router"
...
updateLocale(locale) {
    this.context.store.dispatch(setLocale(locale))
    browserHistory.push("/")
}
```

The url update but the content does not.
With the help of `react-logger` I saw this log : 
`phenomic: PageContainer: replacing by '/' to '/mypage/'`. 
It seems phenomic replace my redirection with my current page.

The workaround is to use `window.location.href = "/"` but there is the uggly "refresh" effect.

Thanks fo your help

### PR DESCRIPTION
**Pull request by xuopled**
Fix #821

### CODE REVIEW

File: docs/content/docs/usage/routing.md [hunk 1]
```diff
@@ -159,3 +159,55 @@ export default (
   </Route>
 )
 ```
+
+## Manipulate history
+
+To create basics links, you need to use the `Link` component from `react-router`
```
Comment (MoOx) [inline docs/content/docs/usage/routing.md]: more ``from "phenomic"`` :)

File: docs/content/docs/usage/routing.md [hunk 2]
```diff
@@ -159,3 +159,55 @@ export default (
...
+
+```
```
Comment (MoOx) [inline docs/content/docs/usage/routing.md]: can you add "js" here?

File: docs/content/docs/usage/routing.md [hunk 3]
```diff
@@ -159,3 +159,55 @@ export default (
   </Route>
 )
 ```
+
+## Manipulate history
```
Comment (MoOx) [inline docs/content/docs/usage/routing.md]: Manipulating

File: docs/content/docs/usage/routing.md [hunk 4]
```diff
@@ -159,3 +159,55 @@ export default (
...
+      {"Homepage"}
+    </Link>
+    // ...
+  </div>
+)
+```
+
+But maybe you need to manipulate history outside of `Link` context.
+
+If you already use `react-router` you know [browserHistory](https://github.com/ReactTraining/react-router/blob/master/docs/guides/Histories.md).
```
Comment (MoOx) [inline docs/content/docs/usage/routing.md]: If you already use `react-router`, you may know [browserHistory

File: docs/content/docs/usage/routing.md [hunk 5]
```diff
@@ -159,3 +159,55 @@ export default (
...
+
+`Phenomic` automatically inject it into `react-router` router history prop.
```
Comment (MoOx) [inline docs/content/docs/usage/routing.md]: Not need for backticks around Phenomic

Comment (MoOx) [inline docs/content/docs/usage/routing.md]: Also it's more "Phenomic injects its own browserHistory instance as a react-router props.

File: docs/content/docs/usage/routing.md [hunk 6]
```diff
@@ -159,3 +159,55 @@ export default (
...
+So instead of using `react-router` `browserHistory` directly, you should use the object exported by `Phenomic`.
+
+```
```
Comment (MoOx) [inline docs/content/docs/usage/routing.md]: add js too here

**Comment by xuopled** [comment]
Thank you for your feedback. I did updates

**Comment by MoOx** [comment]
Thanks!

**Pull request merged**